flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/KafkaTopicPartitionState.java

```diff
@@ -95,6 +95,10 @@ public final boolean isOffsetDefined() {
 		return offset != KafkaTopicPartitionStateSentinel.OFFSET_NOT_SET;
 	}
 
+	public final boolean isSentinel() {
```

Comment:
nit: would `hasSentinelOffset` be a better name here?

Reply:
In new approach method was dropped